src/main/java/by/itechart/api/security/SecurityConfig.java

```diff
@@ -0,0 +1,25 @@
+package by.itechart.api.security;
```

Comment:
Overall, package naming is good. I like it! 
In this particular case it seems logical to me but moving further I think we will have more configuration classes and it would be better to keep them all in the `configuration` package. WDYT?

Reply:
Seems legit! I will change the package level to `configuration -> security`.